File: app/src/main/java/vergecurrency/vergewallet/viewmodel/PinPromptedViewModel.java
package vergecurrency.vergewallet.viewmodel;

import androidx.lifecycle.ViewModel;

import vergecurrency.vergewallet.service.model.PreferencesManager;

public class PinPromptedViewModel extends ViewModel {

	public PinPromptedViewModel(){

	}


	public String getPin() {
		return PreferencesManager.getPin();
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/viewmodel/PinPromptedViewModel.kt
package vergecurrency.vergewallet.viewmodel

import androidx.lifecycle.ViewModel
import vergecurrency.vergewallet.service.model.EncryptedPreferencesManager

class PinPromptedViewModel : ViewModel() {


    val pin: ByteArray
        get() = EncryptedPreferencesManager.pin!!
}
